src/main/java/net/mcreator/ui/modgui/BlockGUI.java

```diff
@@ -104,6 +104,7 @@
 	private ProcedureSelector onRedstoneOff;
 
 	private ProcedureSelector particleCondition;
+	private ProcedureSelector redstoneCondition;

```

Comment:
This is not good name, as this is not condition, but value/integer, consider calling it redstonePowerProvider

Reply:
Done.